pages/common/lorem.md

```diff
@@ -0,0 +1,19 @@
+# lorem
+
+> Create more or less random lorem ipsum text.
+
+- Print the number of words you want:
+
+`lorem -n {{20}}`
+
+- Print 10 lines of Goethe's Faust:
+
+`lorem -l {{10}} --faust`
+
+- Print 5 sentences of Poe's Raven:
+
+`lorem -s {{5}} --raven`
+
+- Print random words from Boccaccio's Decameron for 40 characters:
```

Comment:
I would keep it consistent with the previous descriptions - 

"Print 40 random characters from Boccacio's Decameron".

Reply:
My idea was to show that the order options does not matter.
But the inconsistency might be more confusing than helping.